Fixes #2568 Fix issue with shift and multi tapping

keianhzo opened the pull request:
Fixes #2568 Fix issue with shift and multi tapping

bluemarvin commented:
This patch doesn't work the way I expect. The time out seems to be too long. The double click gets the shift key into caps lock mode. But after that we should just reset since I don't think we need to support a triple click?

keianhzo commented:
@bluemarvin I've rebased and reduced the multitap timeout. Also I have fixed the keyboard refresh, I've seen that we weren't updating both shift keys when multitapping. I think it's ok if multitapping several times keeps it enabled just in case you click more than two times by mistake.
